nix/shell-module.nix

```
@@ -50,16 +52,16 @@ in
 
   scripts = {
     re-up = {
-      description = "re-up";
+      description = "Load compose.yaml into docker (see refresh-compose)";
```

This would be more accurate:

```suggestion
      description = "Builds compose.nix, (re)creates and (re)starts the dev docker containers for Runtime.";
```